### PR TITLE
fix: Allow multiple names to `jx step create pr (docker|regex)`

### LINKED ISSUE #5387: `jx step create pr docker` should allow multiple names.
We should be able to pass multiple names to `jx step create pr docker`, like we can with `jx step create pr chart`.

### LINKED ISSUE #5388: `jx step create pr regex` should allow multiple regexs in a single call
`jx step create pr regex` just allows specifying a single regular expression currently. We should change that to allow multiple regular expressions in a single call.

### PR DESCRIPTION
**Pull request by abayer**
#### Submitter checklist

- [x] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Special notes for the reviewer(s)

/assign @garethjevans 
/assign @cagiti 
/assign @pmuir 

#### Which issue this PR fixes

fixes #5387
fixes #5388

### CODE REVIEW

File: pkg/cmd/step/create/pr/step_create_pr_regex.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func NewCmdStepCreatePullRequestRegex(commonOpts *opts.CommonOptions) *cobra.Com
 		},
 	}
 	AddStepCreatePrFlags(cmd, &options.StepCreatePrOptions)
-	cmd.Flags().StringVarP(&options.Regexp, "regex", "", "", "The regex to use when doing updates")
+	cmd.Flags().StringArrayVarP(&options.Regexps, "regexs", "", make([]string, 0), "The regex to use when doing updates")
```
Comment (garethjevans) [inline pkg/cmd/step/create/pr/step_create_pr_regex.go]: should this stay as `regex` to make the flag backwardly compatible?

Comment (abayer) [reply pkg/cmd/step/create/pr/step_create_pr_regex.go]: Ah, yup, I misunderstood how the array var names worked. Fixing.

**Pull request merged**